modules/relax/models/mixins/hashMap.cfm

```diff
@@ -0,0 +1,25 @@
+<cfscript>
```

Comment:
@lmajano : I know, this looks like a janky mixing, but @evagoras Ran in to a ton of errors on ACF11 and I was able to duplicate the issue on 11 & 2016.  It was really tough to debug.  

Basically, it's the difference returned by using `createObject` on Lucee vs ACF and the way it's used to maintain order in the doc. Unless you use the `putAll` or `put` methods to append to the hash map, the order isn't maintained according to the assembly within the request.

 ACF will throw a key exception if you try to use struct key bracket notation to retrieve a key on a linked hash map - unless you call the init method on the hash map explicitly.  On Lucee, that `init()` is called automatically the first time the object is used.  When I tried to switch everything over to an explicit `init()` on the hashmap, it seemed to put Lucee in to an endless loop.  

The only way I could get it to work across all platforms was to separate the two.